### PR TITLE
IPC Defibrillation & Cult Eyes text changes.

### PR DESCRIPTION
**Pull request by Code611**
# Document the changes in your pull request
**Adds two new dialogues for the defibrillator to alert newbie medics that box head people can't be defibbed or surgically repaired.**

1. **The first one reflects the initial message for whatever is making them unrevivable, but mostly hints the user that surgery can't help an IPC repair damage.** 
toomuchdmg

2. **The second one refers to if the player tries to defib a revivable IPC**
ressuscitation

While yes this may not come into play OFTEN, and the new player might not even care enough to try defibbing them again, hopefully this will stop those rare situations where a player just assumed they must be cloned ((only to find out ipcs cant be cloned))

**Additionally, IPCs now have a unique cult_eyes flavor text to compensate for having a big screen instead of eyes.**
culteyesipc

Everything has been tested, most importantly that the code for hiding cult eyes wasn't botched. 

# Wiki Documentation
Wiki should denote IPC's have a different cult eyes text, even if "BLOOD-RED SWIRLS" isn't obvious enough.
# Changelog
:cl:  
tweak: IPC's now have 2 new unique defibrillation messages.
tweak: IPC's cult "eyes" now detail a monitor as opposed to eyes.
/:cl:

### CODE REVIEW

File: code/game/objects/items/defib.dm [hunk 1]
```diff
@@ -588,10 +588,14 @@
 					failed = span_warning("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Recovery of patient impossible. Further attempts futile.")
 				else if (H.hellbound)
 					failed = span_warning("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Patient's soul unrecoverable.  Further attempts futile.")
+				else if((total_burn >= MAX_REVIVE_FIRE_DAMAGE || total_brute >= MAX_REVIVE_BRUTE_DAMAGE) & (isipc(H))) 
+					failed = span_boldnotice("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Severe chassis damage makes patient recovery impossible via defibrillation or reboot board. Mechanical repairs via tools may result in successful operation.")
 				else if(total_burn >= MAX_REVIVE_FIRE_DAMAGE || total_brute >= MAX_REVIVE_BRUTE_DAMAGE)
 					failed = span_boldnotice("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Severe tissue damage makes recovery of patient impossible via defibrillator. Surgical repair may allow for successful resuscitation.")
```
Comment (tattax) [inline code/game/objects/items/defib.dm]:
```suggestion
				else if(total_burn >= MAX_REVIVE_FIRE_DAMAGE || total_brute >= MAX_REVIVE_BRUTE_DAMAGE)
					failed = span_boldnotice("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed -  Severe [isipc(H) ? "chassis" : "tissue"] damage makes recovery of patient impossible via defibrillator [isipc(H) ? "or reboot board. Mechanical repairs via tools may result in successful operation." : ". Surgical repair may allow for successful resuscitation.")
```

File: code/game/objects/items/defib.dm [hunk 2]
```diff
@@ -588,10 +588,14 @@
...
 				else if(HAS_TRAIT(H, TRAIT_HUSK))
 					failed = span_boldnotice("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Lack of important fluids has rendered the patient body unsurvivable. Application of experimental DNA recovery surgery or an upgraded cloner recommended.")
+				else if (HAS_TRAIT(H, TRAIT_NODEFIB) & isipc(H))
```
Comment (tattax) [inline code/game/objects/items/defib.dm]: just a heads up, & is for checking flags, && is for the and conditonal

File: code/game/objects/items/defib.dm [hunk 3]
```diff
@@ -588,10 +588,14 @@
...
+					failed = span_boldnotice("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Patient's biology is a positronic chassis. Reboot board usage as opposed to defibrillating shocks may prove successful.")
 				else if (HAS_TRAIT(H, TRAIT_NODEFIB))
 					failed = span_boldnotice("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Biology incompatible.")
```
Comment (tattax) [inline code/game/objects/items/defib.dm]:
```suggestion
				else if (HAS_TRAIT(H, TRAIT_NODEFIB))
					failed = span_boldnotice("[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - [isipc(H) ? "Patient's biology is a positronic chassis. Reboot board usage as opposed to defibrillating shocks may prove successful." : "Biology incompatible."]")
```

**Comment by Code611** [comment]
Turns out that the last commit was final; everything works.

**Comment by ilikesaying** [comment]
image

**Comment by Code611** [comment]
> image

erm whoops that was meant to say that ipcs cant be cloned
So not only does the new player see "huh. unrevivable.. I put in big green tube" and then take them to cloner and see they are also unclonable.

**Review by MajManatee — CHANGES_REQUESTED** [review]
Please atomize these changes, then sure.

**Comment by Code611** [comment]
I'll get on it.

**Pull request closed without merging**